### PR TITLE
Engraving search

### PR DESCRIPTION
**Pull request by renatoagds**
Include a new feature that's engraving search.

It allows you to search for two engravings, and see which build fits that one.

image

### CODE REVIEW

File: apps/client/src/app/core/services/builds.service.ts [hunk 1]
```diff
@@ -0,0 +1,492 @@
...
+            name: "Gravity Training Destroyer",
+            stats: [LostArkStat.Specialiazation, LostArkStat.Crit],
+            engravings: [
+                this.getEngravingIdByName("Gravity Training"),
+                this.getEngravingIdByName("Barricade"),
+                this.getEngravingIdByName("Master Brawler"),
+                this.getEngravingIdByName("Spirit Absorption"),
+                this.getEngravingIdByName("Grudge"),
+                this.getEngravingIdByName("Cursed Doll")
+            ]
```
Comment (attrib) [inline apps/client/src/app/core/services/builds.service.ts:478]: Missing scouter build. I have no clue regarding that class.

as of https://maxroll.gg/lost-ark/build-guides/evolutionary-legacy-machinist-raid-guide

Evolutionary Legacy Machinist
Stats: spec, crit
Engravings:
* Evolutionary Legacy
* Adrenaline
* Grudge
* Keen Blunt Weapon
* Barricade
* Raid Captain

Arthetinean Skill Machinist
Stats: crit, swiftness
Engravings:
* Arthetinean Skill 
* Adrenaline
* Grudge
* Keen Blunt Weapon
* Raid Captain
* Mass Increase

**Comment by attrib** [comment]
How about a 3rd optional select box for one stat. So it filters out all combos which have a wrong stat?

**Comment by Supamiu** [comment]
Wow that's an amazing PR ! Keep it up, it'll be really cool once merged 👍

File: apps/client/src/app/core/services/builds.service.ts (hunk 1 above)
Comment (renatoagds) [reply apps/client/src/app/core/services/builds.service.ts:478]: Is missing Arcanist build too. Will add it

Comment (renatoagds) [reply apps/client/src/app/core/services/builds.service.ts:478]: Done https://github.com/Supamiu/Lostark-helper/pull/52/commits/d918d526e7bddc89886bb94957fd91a96b4a25f2

**Review by attrib — APPROVED** [review]
Looks good now.

**Review by Supamiu — APPROVED** [review]

**Pull request merged**